### DISCUSSION TITLE
Websocket connection to "url here" failed:

### DISCUSSION THREAD

**Discussion by veerack** [fastapi/fastapi#6576] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
root_router = APIRouter()

class SocketManager:
    def __init__(self):
        self.active_connections: List[(WebSocket, str)] = []

    async def connect(self, websocket: WebSocket, user: str):
        await websocket.accept()
        self.active_connections.append((websocket, user))

    def disconnect(self, websocket: WebSocket, user: str):
        self.active_connections.remove((websocket, user))

    async def broadcast(self, data):
        for connection in self.active_connections:
            await connection[0].send_json(data)    

manager = SocketManager()

@root_router.websocket("/chat/ws")
async def chat(websocket: WebSocket):
    sender = websocket.cookies.get("X-Authorization")
    if sender:
        await manager.connect(websocket, sender)
        response = {
            "sender": sender,
            "message": "got connected"
        }
        await manager.broadcast(response)
        try:
            while True:
                data = await websocket.receive_json()
                await manager.broadcast(data)
        except WebSocketDisconnect as e:
            print(e)
            manager.disconnect(websocket, sender)
            response['message'] = "left"
            await manager.broadcast(response)
```

### Description

Trying to connect to the websocket, i receive the following:
`WebSocket connection to 'wss://url:8000/path' failed`

i searched for 3h on internet, and didn't found why is not working.
totally sure my setup is right, running the app on Ubuntu 20.04 (Linux Distro), with uvicorn.

### Operating System

Linux

### FastAPI Version

lastest one

### Python Version

3.9.0

**Comment by Honda-a** [comment]
could you please alaborate more on what you want to do and how its not working?
if you could show your HTML/JS file where you send the websocket request

**Comment by veerack** [comment, marked as answer]
> could you please alaborate more on what you want to do and how its not working?
> if you could show your HTML/JS file where you send the websocket request

thanks for the reply, but i actually solved, it was an issue on my end with my reverse proxy not allowing websockets >.>
